site/2017/10/24/odm-1-2-0-and-1-1-7.rst

```diff
@@ -0,0 +1,81 @@
+Doctrine MongoDB ODM 1.2.0 and 1.1.7 Released
+=============================================
+
+We are happy to announce the immediate availability of Doctrine MongoDB ODM
+`1.2.0 <https://github.com/doctrine/mongodb-odm/releases/tag/1.2.0>`_ and
+`1.1.7 <https://github.com/doctrine/mongodb-odm/releases/tag/1.1.7>`_.
+
+MongoDB ODM 1.1.7
+-----------------
+
+Notable fixes may be found in the
+`changelog <https://github.com/doctrine/mongodb-odm/blob/1.1.x/CHANGELOG-1.1.md#117-2017-10-23>`_.
+A full list of issues and pull requests included in this release may be found in the
+`1.1.7 milestone <https://github.com/doctrine/mongodb-odm/issues?q=milestone%3A1.1.7>`_.
+
+What is new in 1.2.0?
+---------------------
+
+Doctrine MongoDB ODM 1.2.0 comes with the following new features:
+
+- The ``readOnly`` mapping option allows you to map immutable documents.
+- With ``slaveOkay`` being deprecated, you can now specify a ``readPreference``
+  for your documents. This ``readPreference`` will automatically be applied to
+  all queries run on the collection.
+- With ``dbRef`` only offering limited support in aggregation pipeline queries,
+  there's a new reference storage strategy called ``ref`` that stores references
+  as objects, allowing you to use it with discriminators and aggregation
+  pipeline queries.
+- When mapping inverse references, you may now specify fields that will be
+  primed when the reference is resolved. This saves you from writing a dedicated
+  repository method for the sole purpose of priming references.
+- The ``odm:schema:validate`` command validates the document mapping to help you
+  spot mistakes.
+- A new builder for aggregation pipeline queries, including support for
+  marshalling results into read-only documents.
+- Query result documents that can be used to hydrate results from an aggregation
+  pipeline query. These documents cannot be written back to the database.
+
+Upgrading to 1.2.0
+------------------
+
+The new version requires PHP 5.6. Running on PHP 7+ requires the use of a
+polyfill for the legacy driver, e.g. `mongo-php-adapter <https://github.com/alcaeus/mongo-php-adapter>`_.
+When running on PHP 5, MongoDB ODM requires version 1.6.7 or newer of the legacy
+MongoDB driver.
```

Comment:
Is it worth adding "(or a polyfill library)" to encourage PHP 5.6 users to also use your library and the new driver?

Reply:
I've refrained from suggesting the adapter to people running PHP 5.6 because of its performance impact. Unless someone runs into an issue in ext-mongo that would be fixed by using the adapter and ext-mongodb, I still wouldn't recommend it.